src/app/pre-sale/providers/presale-provider.tsx

```diff
@@ -29,19 +29,18 @@ export function usePresaleData(symbol: string): PresaleData {
   )
 
   useEffect(() => {
-    // FIXME: use hyETH (presale token)
-    const token = MetaverseIndex
+    if (!presaleToken) return
     const fetchTvl = async () => {
       try {
         const indexApi = new IndexApi()
-        const res = await indexApi.get(`/${token.symbol}/marketcap`)
+        const res = await indexApi.get(`/${presaleToken.symbol}/marketcap`)
         setTvl(res.marketcap)
       } catch (err) {
         console.log('Error fetching tvl', err)
       }
     }
     fetchTvl()
-  }, [symbol])
+  }, [presaleToken])
```

Comment:
hmm, was this copied from somewhere else? (wondering why `symbol` is mentioned in the deps array)

Reply:
The hook `export function usePresaleData(symbol: string): PresaleData {` is using a symbol as in input. Before that changes were based on the symbol but now we have the `preSaleToken` object available.